docs/build-apps/build-items.md

```diff
@@ -9,6 +9,42 @@ ms.date: 09/19/2024
 Build items control how .NET for iOS, Mac Catalyst, macOS, and tvOS
 application or library projects are built.
 
+## PartialAppManifest
+
+`PartialAppManifest` can be used to add additional partial app manifests that
+will be merged with the main app manifest (Info.plist).
+
+Any values in the partial app manifests will override values in the main app
+manifest unless the `Overwrite` metadata is set to `false`.
```

Comment:
Should this be "`Overwrite` attribute" instead of metadata?

Reply:
No, it's metadata: https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-items?view=vs-2022#item-metadata